website/docs/r/cas_certificate.html.makrdown

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Please add a link in the file `website/alicloud.erb` for this file.

Reply:
done